src/js/components/Calendar/StyledCalendar.js

```diff
@@ -44,6 +44,17 @@ const StyledCalendar = styled.div.withConfig(styledComponentsConfig)`
   ${(props) => sizeStyle(props)}
   ${(props) => props.responsive && responsiveSizeStyle(props)}
   ${(props) => props.theme.calendar && props.theme.calendar.extend}
+
+  ${(props) =>
+    props.disabled &&
+    `
+    opacity: 0.5;
```

Comment:
Whatever the disabled treatment is, it should probably come from the theme and not be a hardcoded opacity